release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -396,6 +625,14 @@ following command will also be deprecated at that time:
 For more information on the upcoming new Operator bundle format and Operator
 Package Manager CLI (`opm`), see the link:https://docs.okd.io/latest/operators/understanding_olm/olm-understanding-olm.html#olm-new-bundle-opm_olm-understanding-olm[upstream OKD documentation].
 
+[id="ocp-4-4-drop-high-granularity-request-duration-buckets"]
+==== Drop high granularity request duration buckets in Prometheus
+
+You should drop high granularity request duration buckets in Prometheus, which
```

Comment:
Only note is, we do drop them already. So:
```suggestion
We dropped high granularity request duration buckets in Prometheus, which
```

Reply:
ah, got it. Thanks for reviewing @lilic! I'll make that change and get this into the release notes.